Makefile.devel

```diff
@@ -58,8 +58,17 @@ always-build-chplspell-venv: FORCE
 	$(MAKE) chplspell-venv; \
 	fi
 
-test: FORCE
-	cd test && start_test
+test: spectests FORCE
+	cd examples && start_test
+
+libchplcomp-docs: FORCE
+	@cd compiler/next && $(MAKE) -f Makefile.help libchplcomp-docs
```

Comment:
You could do the same thing for these three as I described for `Makefile:libchplcomp`.